src/content/concepts/index.md

```diff
@@ -10,13 +10,16 @@ contributors:
   - TheDutchCoder
   - adambraimbridge
   - EugeneHlushko
+  - jeremenichelli
 ---
 
-At its core, *webpack* is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it recursively builds a _dependency graph_ that includes every module your application needs, then packages all of those modules into one or more _bundles_.
+At its core, **webpack** is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it internally builds a _dependency graph_ which maps every module your project needs, then it generates one or more _bundles_.
```

Comment:
maybe its me but sounds more natural as `project needs and generates one...`

Reply:
Agree that sounds better like that. Just pushed the changed.